settings.js

```diff
@@ -203,8 +202,7 @@ function _migrateToV1(migrationSettings, extensionSettings) {
 
         if (!app.should_show() ||
             addedItems.has(appId) ||
-            appsInsideFolders.has(appId) ||
-            appId.startsWith(EOS_LINK_PREFIX))
```

Comment:
I'm a little confused: this seems to only be used at migration time. But that means it has almost no effect, since on subsequent logins all these desktop files will just show up on whatever page as normal, right? Certainly they did on my computer way back. :-)

So I think we can go ahead and merge this change since I don't think it will affect the observed behaviour.

Reply:
Yes, I was also confused and wondering if there was some other place special handling it but could not find any in shell or other extensions.